packages/shared/src/components/cards/v1/ActionButtons.tsx

```diff
@@ -102,26 +103,23 @@ export default function ActionButtons({
             }
           >
             <Button
-              className="pointer-events-auto hover:text-theme-color-avocado"
+              className={classNames(
+                'pointer-events-auto',
```

Comment:
Missing `hover:text-theme-color-avocado`

```suggestion
                'pointer-events-auto hover:text-theme-color-avocado',
```

Reply:
We don't need it, since it is automatically applied from the `Button` component.